### PR TITLE
bug 1797797: fix(machineconfig.crd.yaml): add passwd schema

### PR DESCRIPTION
**Pull request by ericavonb**
Cherry-pick of #1234

### CODE REVIEW

File: manifests/machineconfig.crd.yaml [hunk 1]
```diff
@@ -34,3 +34,202 @@ spec:
...
+                              description: "items describes unit dropin"
+                              type: object
+                              properties:
+                                contents:
+                                  description: "contents is the contents of the drop-in"
+                                  type: string
+                                name:
+                                  description: "name is the name of the drop-in. This must be suffixed with '.conf'."
+                                  type: string
+          kargs:
```
Comment (sinnykumari) [inline manifests/machineconfig.crd.yaml]: I don't think we support kargs in 4.1. Not sure about fips though

Comment (runcom) [reply manifests/machineconfig.crd.yaml]: both aren't supported in 4.1

Comment (ericavonb) [reply manifests/machineconfig.crd.yaml]: thanks for catching that. removed.

**Comment by ericavonb** [comment]
/close

per conversation with @ashcrow, we won't be backporting this to 4.1

**Pull request closed without merging**